### DISCUSSION TITLE
How to turn off the servo motors using MoveIt/ROS

### DISCUSSION THREAD

**Discussion by SergeAlhalbi** [Yaskawa-Global/motoros2#320] [Q&A]
How can I turn off the servo motors using MoveIt/ROS? I am aware of the /stop_traj_mode service, but it doesn't seem to turn off the servos. It appears to just stop the trajectory while the robot is executing one. I'm looking for a service similar to /start_traj_mode, but on the contrary, one that turns the servos off. I appreciate your help. Thanks!

**Comment by gavanderhoorn** [comment]
Please see #319.
